step-api/src/main/java/org/jenkinsci/plugins/workflow/structs/DescribableHelper.java

```diff
@@ -382,7 +394,8 @@ static ParameterType of(Type type) {
 
     public static final class ArrayType extends ParameterType {
         private final ParameterType elementType;
-        ArrayType(ParameterType elementType) {
+        ArrayType(Class<?> actualClass, ParameterType elementType) {
```

Comment:
Hrm, actually, we've got `return new ArrayType(Collection.class, of(((ParameterizedType) type).getActualTypeArguments()[0]));` in `ParameterType.of` - I think we still need this?